tests/test_format.py

```diff
@@ -135,12 +141,6 @@ def test_public_key_to_address_incorrect_length(self):
         with pytest.raises(ValueError):
             public_key_to_address(PUBLIC_KEY_COMPRESSED[:-1])
 
-    def test_public_key_to_address_test_compressed(self):
-        assert public_key_to_address(PUBLIC_KEY_COMPRESSED, version='test') == BITCOIN_ADDRESS_TEST_COMPRESSED
-
-    def test_public_key_to_address_test_uncompressed(self):
-        assert public_key_to_address(PUBLIC_KEY_UNCOMPRESSED, version='test') == BITCOIN_ADDRESS_TEST
-
```

Comment:
These pass in Travis, for example: https://travis-ci.org/github/pybitcash/bitcash/jobs/731953937